Self-Study/Algorithms_Answers.md

```diff
@@ -1 +1,67 @@
-Add your answers to the Algorithms exercises here.
\ No newline at end of file
+Add your answers to the Algorithms exercises here.
+
+```
+a)  a = 0
+    while (a < n * n * n):
+      a = a + n * n
+```
+
+The above algorithm is `O(n^2)`.
+if `n == 2`,  the loop will run until `a == 8`.
+In this expression: `a = a + n * n` `a` is set to `4`.
+The second time around `a` will become `8`. `4 + 2 * 2 == 8`
+The expression `a + n * n` is only increments `a` by half the required number.
```

Comment:
this line is true, but your answer doesn't follow from this statement (the answer given here is wrong.) 
try testing this for differnet values of n
what is the number of operations executed for:
n = 2
n = 5
n = 10
n = 100